PIPELINE_TEMPLATES.md

```diff
@@ -485,6 +494,56 @@ A couple key things to note here:
    dependencies. Note that the stage `finalWait` stage only depends on 
    `buildChrome.publishTarget`, rather than every stage defined by the Partial.
 
+# Imports
+
+Importing is supported at both the Template and Configuration level and can be
+used to break up a large files into more digestable units, or to share common
+reusable components across multiple Templates. Unlike a Template hierarchy,
+imported Templates _cannot_ define Stages.
+
+Imported Templates share the same file structure as a Template, but some fields
+become ignored:
+
+* `source`: Imports cannot inherit from other templates or other Imports.
+* `imports`: Imports cannot include other Imports.
```

Comment:
I'm curious about why this wouldn't be allowed (not sure if it would be a good idea, but I'm imagining someone might want to put together a module that's composed of other modules).

Reply:
No technical reason except limiting scope of the feature. Definitely some niceities in allowing recursive imports, though.